### ISSUE TITLE
Can´t move miniature video of Youtube Min VId

### ISSUE THREAD

**Issue by johan0320** [meandavejustice/min-vid#1108]
Hey guys!
I like this idea so much!!
but, i can´t move or adjust measurements of **MinVId**. The image is in the middle of the upper left corner of my screen and it does not allow me to change to the next or previous video.

**Comment by SoftVision-CarmenFat** [comment]
Hi @johan0320,
	
Thanks for reporting this.
	
I think you might have encountered issue [#842](https://github.com/meandavejustice/min-vid/issues/842) where sending a Playlist to the mini player moves it on the top of the screen and hides half of it.
Can you please confirm this?

**Comment by johan0320** [comment]
Yes, I think so.

**Comment by johan0320** [comment]
But today I tried again and I did not have the same problem...:-D

**Comment by johan0320** [comment]
apparently it is random, as it has failed again

**Comment by MinionAttack** [comment]
I came here to post the same. But I've discovered that if you move the mouse slowly and drag the mind vid to the position you want it works.

The Pop-up get stuck when I try to move it faster. Doing slowly works fine :/

It seems like loses the focus of the mouse. I've checked it clicking and draging it maintaining the mouse clicked and clicking and moving without maintaing the mouse clicked.

Regards!
